kernel/src/kernel_predicates/mod.rs

```diff
@@ -639,22 +639,20 @@ impl<R: ResolveColumnAsScalar> KernelPredicateEvaluator for DefaultKernelPredica
 pub(crate) trait DataSkippingPredicateEvaluator {
     /// The output type produced by this predicate evaluator
     type Output;
-    /// The type of min and max column stats
-    type TypedStat;
-    /// The type of nullcount and rowcount column stats
-    type IntStat;
+    /// The type of min column stats
```

Comment:
nit: (or could just say "the type of column stats")
```suggestion
    /// The type of min/max/nullcount/rowcount column stats
```